docs/source/deepxde.rst

```diff
@@ -4,8 +4,47 @@ DeepXDE
 
 To do. Explain the theory for inverse problems, and add references.
 
+
+Installation 
+============
+
+
+**Requirements**
+
+To run **DeepXDE** in your local machine or in a computer cluster is necessary install **TensorFlow** as a backend library.
+In order to do that, first you need to install **TensorFlow** using the command line: ::
+
+$ pip install TensorFlow  #Using pip as a installer
+$ conda install TensorFlow #Using conda as a installer
+
+
+The next step is to **install the DeepXDE library** with one of these commands: ::
+
+$ pip install deepxde #Using pip as installer
```

Comment:
Check exactly which other libraries have to be installed (tensorflow probability or something maybe).

Reply:
Got it